games.txt

```diff
@@ -217,6 +217,7 @@ rtcw|Return to Castle Wolfenstein|quake3|port_query=27960
 rfactor|rFactor|rfactor|port=34397,port_query_offset=-100
 ricochet|Ricochet|valve|port=27015
 riseofnations|Rise of Nations|gamespy1|port_query=6501
+ror2|Risk of Rain 2 (2020)|valve|27015,port_query_offset=1
```

Comment:
should probably be `port=27015,port_query_offset=1` instead of `27015,port_query_offset=1`

Reply:
Well spotted! Fixed.